doc/install.rst

```diff
@@ -210,3 +211,29 @@ Ginga.
 
 After installing Anaconda, open the Anaconda Prompt and follow instructions
 under :ref:`install_generic` via ``conda``.
+
+=============
+Running tests
+=============
+
+#. Install the following packages::
+
+    $ pip install pytest pytest-astropy-header
+
+#. Run the tests using `pytest`::
+
+    $ pytest
+
+======================
+Building documentation
+======================
+
+#. Install the following packages::
+
+    $ pip install sphinx-astropy sphinx_rtd_theme
```

Comment:
Maybe like this?
```suggestion
    $ pip install -e .[docs]
```

Reply:
Nice! Tested?

Reply:
Looks like we also use `recommended` packages for RTD build

https://github.com/ejeschke/ginga/blob/c399f2e2b3bd38857b7a15804a2430e3d0cd7180/.readthedocs.yml#L14-L19